usim_pytest/test_types/test_pipe.py

```diff
@@ -0,0 +1,60 @@
+from usim import time, Pipe, Scope
+
+from ..utility import via_usim
+
+
+async def aenumerate(aiterable, start=0):
+    count = start
+    async for item in aiterable:
+        yield count, item
+        count += 1
+
+
+class TestPipe:
+    @via_usim
+    async def test_transfer_uncontested(self):
```

Comment:
Your grammar correction is really picky ;)

```suggestion
    async def test_transfer_uncongested(self):
```